doc/release-process.md

```diff
@@ -207,16 +207,25 @@ cat "$VERSION"/*/all.SHA256SUMS.asc > SHA256SUMS.asc
 
 
 - Upload to the bitcoincore.org server (`/var/www/bin/bitcoin-core-${VERSION}`):
-    1. The contents of `./bitcoin/guix-build-${VERSION}/output`, except for
+    1. The contents of each `./bitcoin/guix-build-${VERSION}/output/${HOST}` directory, except for
```

Comment:
Nit: I personally like stating directory names with a traling `/` when possible like here, so code readers can easily recognize that it is a directory, in this case "each `./bitcoin/guix-build-${VERSION}/output/${HOST}/` directory". In line 226 this is done with `/output/`

Reply:
Done